packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -126,6 +126,9 @@ function getNumberOfAnnotations(
  * @param annotationUID - The unique identifier for the annotation.
  */
 function removeAnnotation(annotationUID: string): void {
+  if (!annotationUID) {
+    return;
```

Comment:
I think we should throw meaningful error here instead of return

Reply:
We already have a check below that handles this situation, I just wanted a specific on if the annotationUID isn't defined so it is really obvious, but I don't want to change the behaviour that was pre-existing that you can remove something that doesn't exist.